src/style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: bold;
+  font-size: 100px;
+  color: white;
+  box-sizing: border-box;
+  margin: 0;
+
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.box {
+  display: flex;
+  height: 300px;
+  min-width: 300px;
+  flex-basis: 100%;
+  flex-shrink: 0;
+  background-color: rgb(0%, 0%, 0%);
+  justify-content: center;
+  align-items: center;
+}
+
+.box-1 {
+  background-color: rgb(100%, 0%, 0%);
+}
+
+.box-2 {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.box-3 {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.box-4 {
+  background-color: rgb(40%, 0%, 0%);
+}
+
+.box-5 {
+  background-color: rgb(20%, 0%, 0%);
+}
+
+@media (min-width: 600px) {
+  .box {
+    flex-basis: 50%;
+  }
+
+  .box-3 {
+    order: 2;
+  }
+
+  .box-4 {
+    order: 1;
+  }
+
+  .box-5 {
+    order: 3;
+  }
+
+  .box-6 {
+    order: 4;
+  }
+}
+
+@media (min-width: 900px) {
+  .box {
+    flex-basis: 33.3%;
```

Comment:
if you add 3 of those, you will only end up covering 99.9%. it's best to let the math figure it out.
100%/3